code/iaas/agent-instance/src/main/resources/META-INF/cattle/system-services/agent-instance-defaults.properties

```diff
@@ -9,7 +9,7 @@ agent.instance.start.items.increment.extra=
 agent.instance.services.base.items=${agent.instance.start.items.apply},${agent.instance.start.items.increment},${agent.instance.services.base.items.extra}
 agent.instance.services.base.items.extra=
 
-agent.instance.services.processes=nic.activate,instancelink.update,instancelink.activate,hostipaddressmap.activate,${agent.instance.services.processes.extra}
+agent.instance.services.processes=nic.activate,instancelink.update,instancelink.activate,hostipaddressmap.activate${agent.instance.services.processes.extra}
```

Comment:
Sorry, Darren, removed the "," accidentally when I removed "port.activate"/"port.deactivate" (added them previously, and then realized that it shouldn't be there). I'll roll back the changes for this file.